dali/operators/reader/loader/video/frames_decoder_test.cc

```diff
@@ -121,7 +124,44 @@ TEST_F(FramesDecoderTest, VariableFrameRateCpu) {
     this->CompareFrames(frame.data(), this->GetVfrFrame(1, 0), file.FrameSize());
 }
 
-TEST_F(FramesDecoderTest, VariableFrameRateGpu) {
+TEST_F(FramesDecoderTest_CpuOnlyTests, InvalidPath) {
+    std::string path = "invalid_path.mp4";
+
+    try {
+        FramesDecoder file(path);
+    } catch (const DALIException &e) {
+        EXPECT_TRUE(strstr(
+            e.what(),
+            make_string("Failed to open video file at path ", path).c_str()));
```

Comment:
How about
```EXPECT_THROW(FramesDecoder(path), DALIException)```
?
I don't think you can match the error message, so up to you

Reply:
Yeah, I wanted to make sure that the message is the one I want and it reflects what happen, so I'm going to leave it for now.
If this keeps reoccurring we can think on some solution that would do both.